src/averageAges.js

```diff
@@ -37,7 +52,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const everyWomen = people.filter(person =>
+    person.sex === 'f');
```

Comment:
The condition is short enough, you can write it in one line. Or, if you want to do it on a new line, use parentheses as required in checklist point 2.
```suggestion
  const everyWomen = people.filter(person => person.sex === 'f');
```

Reply:
i will fix it